src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrStates = [];
+  let stateCopy = { ...state };
+
+  for (const obj of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
its not obj, it's action inside actions